source/common/json/config_schemas.cc

```diff
@@ -1205,6 +1205,10 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
         "enum" : ["http2"]
       },
       "http_codec_options" : {"type" : "string"},
+      "dns_resolvers": {
+        "type" : "array",
```

Comment:
Fixed in https://github.com/lyft/envoy/pull/1018/commits/6137332c35b28def98741f74a891365c483293a6.